tools/dds/py-dds-server/python.cpp

```diff
@@ -233,6 +297,21 @@ PYBIND11_MODULE(NAME, m) {
             return os.str();
         } );
 
+    //py::class_< raw_device_info, std::shared_ptr< raw_device_info > >( raw, "device_info" )
```

Comment:
?

Reply:
I left it in there because I didn't want to lose the code... will remove later if not needed